stratum/hal/lib/common/gnmi_publisher.h

```diff
@@ -140,6 +140,16 @@ class GnmiPublisher {
   // the switch and cleaning-up.
   virtual ::util::Status UnregisterEventWriter() LOCKS_EXCLUDED(access_lock_);
 
+  // IPsec is a special use-case, since yang tree nodes are not initialized and
```

Comment:
I recommend moving this comment (which answers the question "why?") to the method body and replacing it with a shorter comment that says what the method _does_.

I'd also suggest renaming the method. (See comment on the method body.)